input/pagecontent/conformance.md

```diff
@@ -1,18 +1,8 @@
-An [AU eRequesting Placer](ActorDefinition-au-erequesting-actor-placer.html):
+Each AU eRequesting actor as defined in [Actors and Capabilities](capability-statements.html):
 - **SHALL** implement the [mandatory](#mandatory-elements) and/or [Must Support](#must-support-and-obligation) requirements for the elements defined as such in the implemented AU eRequesting profiles.
 - **SHALL** implement the required REST API interactions and search parameters as defined in the [AU eRequesting Placer Capability Statement](CapabilityStatement-au-erequesting-placer.html#resourcesSummary1).
 
-An [AU eRequesting Filler](ActorDefinition-au-erequesting-actor-filler.html):
-- **SHALL** implement the [mandatory](#mandatory-elements) and/or [Must Support](#must-support-and-obligation) requirements for the elements defined as such in the implemented AU eRequesting profiles.
-- **SHALL** implement the required REST API interactions and search parameters as defined in the [AU eRequesting Filler Capability Statement](CapabilityStatement-au-erequesting-filler.html#resourcesSummary1).
-
-An [AU eRequesting Patient](ActorDefinition-au-erequesting-actor-patient.html):
-- **SHALL** implement the [mandatory](#mandatory-elements) and/or [Must Support](#must-support-and-obligation) requirements for the elements defined as such in the implemented AU eRequesting profiles.
-- **SHALL** implement the required REST API interactions and search parameters as defined in the [AU eRequesting Patient Capability Statement](CapabilityStatement-au-erequesting-patient.html#resourcesSummary1).
-
-An [AU eRequesting Server](ActorDefinition-au-erequesting-actor-server.html):
-- **SHALL** implement the [mandatory](#mandatory-elements) and/or [Must Support](#must-support-and-obligation) requirements for the elements defined as such in the implemented AU eRequesting profiles.
-- **SHALL** implement the required REST API interactions and search parameters as defined in the [AU eRequesting Server Capability Statement](CapabilityStatement-au-erequesting-server.html#resourcesSummary1).
+Additionally, the [AU eRequesting Server](ActorDefinition-au-erequesting-actor-server.html) Actor:
 - **SHALL** specify the implemented resource types, associated AU eRequesting profiles, REST API interactions and search parameters in the server's CapabilityStatement.
```

Comment:
This is slightly different to FHIR-46806 resolution description, please update to match.